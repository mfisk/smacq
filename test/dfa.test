#!/bin/sh

bin/smacqq -m <<EOF 
print -v, seq, ack, srcip, dstip, srcport, dstport, ts, previous.seq, previous.ts from dfa($TEST/dfa.test.dfa) from pcapfile($TEST/0325@1112-snort.pcap)
EOF
