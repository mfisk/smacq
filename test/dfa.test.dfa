START FIRST seq
FIRST STOP seq == previous.seq
