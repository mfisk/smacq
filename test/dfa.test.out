seq = 3593444674	ack = 1729801	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1052	dstport = 139	ts = 2000-03-25T12:15:28.021391-07:00	previous.seq = 3593444674	previous.ts = 2000-03-25T12:15:28.021223-07:00
seq = 3593509274	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1053	dstport = 21	ts = 2000-03-25T12:15:28.518317-07:00	previous.seq = 3593509274	previous.ts = 2000-03-25T12:15:28.102582-07:00
seq = 3593571492	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1054	dstport = 80	ts = 2000-03-25T12:15:28.518393-07:00	previous.seq = 3593571492	previous.ts = 2000-03-25T12:15:28.103886-07:00
seq = 3593755013	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1057	dstport = 111	ts = 2000-03-25T12:15:28.620389-07:00	previous.seq = 3593755013	previous.ts = 2000-03-25T12:15:28.129182-07:00
seq = 3593800672	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1058	dstport = 79	ts = 2000-03-25T12:15:28.620464-07:00	previous.seq = 3593800672	previous.ts = 2000-03-25T12:15:28.131837-07:00
seq = 3593845018	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1059	dstport = 53	ts = 2000-03-25T12:15:28.620554-07:00	previous.seq = 3593845018	previous.ts = 2000-03-25T12:15:28.134177-07:00
seq = 3593649239	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1055	dstport = 25	ts = 2000-03-25T12:15:28.620627-07:00	previous.seq = 3593649239	previous.ts = 2000-03-25T12:15:28.120922-07:00
seq = 3593700394	ack = 0	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1056	dstport = 110	ts = 2000-03-25T12:15:28.620701-07:00	previous.seq = 3593700394	previous.ts = 2000-03-25T12:15:28.126184-07:00
seq = 3594727177	ack = 1729878	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1061	dstport = 139	ts = 2000-03-25T12:15:31.422384-07:00	previous.seq = 3594727177	previous.ts = 2000-03-25T12:15:31.422224-07:00
seq = 3594889132	ack = 1729895	srcip = 192.168.0.54	dstip = 192.168.0.200	srcport = 1062	dstport = 135	ts = 2000-03-25T12:15:31.774992-07:00	previous.seq = 3594889132	previous.ts = 2000-03-25T12:15:31.768626-07:00
